runtime/src/lib.rs

```diff
@@ -278,7 +438,6 @@ pub type SignedExtra = (
     frame_system::CheckTxVersion<Runtime>,
     frame_system::CheckGenesis<Runtime>,
     frame_system::CheckEra<Runtime>,
-    frame_system::CheckMortality<Runtime>,
```

Comment:
The error I had in polkadot.js mentioned this check mortality thing, is it something that got removed from the template recently? Do you know what it means to not have it?

Reply:
I'm not 100% sure, I changed it since all templates were using it, it[ appears to be a rename](https://github.com/paritytech/substrate/blob/c939ceba381b6313462d47334f775e128ea4e95d/frame/system/src/lib.rs#L125)